content/compute-app/non-functional-requirements/readme.md

```diff
@@ -0,0 +1,156 @@
+# Overview
+
+During architecture reviews and discussions, FTA engineers work with many different types of customers. We can only be of assistance when we really understand what customers' requirements are, and what they're trying to achieve with a solution design or architecture. However, many customers struggle to understand the importance of their non-functional requirements, and either have absolutely no context to provide such requirements or will make up technical requirements that don't really align with business requirements.
+
+In this FTA Live session, we want to help customers to understand some of the key non-functional requirements we pay attention to, and set them up to have the right conversations internally so they can collect their requirements from the right stakeholders.
+
+## Intended audience
+
+- Solution architects, and especially accidental architects (i.e. those who might not have a lot of formal experience with architecture but are responsible for designing/architecting a solution).
+- Product owners/product managers and other business stakeholders who are ultimately accountable for requirements.
```

Comment:
Solution architects may need to educate their business stakeholders about tradeoffs since a business stakeholder will always say 100% SLA or 0 RPT/RTO unless they know the costs and complexity costs involved. In other words, deriving reasonable metrics requires a "negotiation" between technologists and business folks so that there is a shared understanding of why response SLA of 10ms is required if customers using the solution don't care.

Reply:
That's a great point, and we'll adjust the wording to make it clear that this is a negotiation.

Reply:
Updated!